lib/trestle/form/fields/collection_radio_buttons.rb

```diff
@@ -4,13 +4,14 @@ module Fields
       class CollectionRadioButtons < Field
         include RadioButtonHelpers
 
-        attr_reader :collection, :value_method, :text_method, :html_options
+        attr_reader :collection, :value_method, :text_method, :html_options, :block
 
         def initialize(builder, template, name, collection, value_method, text_method, options={}, html_options={}, &block)
           super(builder, template, name, options, &block)
```

Comment:
`block` gets passed (and then assigned to `@block` in `Field`) via this super call. What issue are you seeing that this fixes?

Reply:
The issue was that the block wasn't being called. Turns out I was looking at an older (release) version where the `if block` logic wasn't even implemented.